code/workspaces/web-app/src/components/app/__snapshots__/Message.spec.js.snap

```diff
@@ -1,25 +1,39 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
 exports[`Message renders correct snapshot 1`] = `
-<div
-  className="Message-message-1"
->
-  <ErrorOutlineIcon
-    className="Message-icon-3"
-  />
+<div>
   <div
-    className="Message-text-2"
+    class="Message-message-1"
   >
-    <mockConstructor>
-      Test message
-    </mockConstructor>
+    <svg
```

Comment:
I'd be tempted to mock ErrorOutline here as you don't really care what the contents of the svg are.